src/Symfony/Component/Scheduler/Messenger/ScheduledStamp.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler\Messenger;
+
+use Symfony\Component\Messenger\Stamp\NonSendableStampInterface;
+
+/**
+ * @experimental
+ */
+final class ScheduledStamp implements NonSendableStampInterface
```

Comment:
maybe we don't need this stamp since we don't use it?

Reply:
@upyx Do you had a use case in mind for that stamp?

Reply:
It implements NonSendableStampInterface which a mark for Messnger to handle message but not route to the default queue.